docs/source/releases/v1.6.rst

```diff
@@ -19,10 +19,10 @@ Table of contents:
 Compatibility
 -------------
 
-Oscar 1.6 is compatible with Django 1.8, 1.10 and 1.11 as well as Python 2.7,
-3.3, 3.4, 3.5 and 3.6. Support for Django 1.9 is no longer officialy supported
-since it is longer supported by Django (end of life).
+Oscar 1.6 is compatible with Django 1.11 and 2.0 as well as Python 2.7,
+3.4, 3.5 and 3.6.
 
+Official support for Django 1.8, 1.9 and 1.10 has been dropped.
```

Comment:
Hm, not true yet for tox.

Reply:
Yes, but these are release notes for 1.6, which will when we release it drop support for Django 1.10 and below?

Reply:
Yes, but we haven't yet - let's firstly drop it and then add note.